src/content/learn/react-developer-tools.md

```diff
@@ -1,33 +1,33 @@
 ---
-title: React Developer Tools
+title: Інструменти розробки React
 ---
 
 <Intro>
 
-Use React Developer Tools to inspect React [components](/learn/your-first-component), edit [props](/learn/passing-props-to-a-component) and [state](/learn/state-a-components-memory), and identify performance problems.
+Використовуйте інструменти розробки React (_React Developer Tools_) для інспектування [компонентів](/learn/your-first-component) React, редагування їх [пропсів](/learn/passing-props-to-a-component) і [стану](/learn/state-a-components-memory), а також для виявлення проблем з продуктивністю.
 
 </Intro>
 
 <YouWillLearn>
 
-* How to install React Developer Tools
+* Як встановити інструменти розробки React
 
 </YouWillLearn>
 
-## Browser extension {/*browser-extension*/}
+## Розширення браузера {/*browser-extension*/}
 
-The easiest way to debug websites built with React is to install the React Developer Tools browser extension. It is available for several popular browsers:
+Найпростіший спосіб налагодження вебсайтів, створених за допомогою React — встановити розширення браузера "Інструменти розробки React" (_React Developer Tools_). Воно доступне для декількох популярних браузерів:
 
-* [Install for **Chrome**](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)
-* [Install for **Firefox**](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/)
-* [Install for **Edge**](https://microsoftedge.microsoft.com/addons/detail/react-developer-tools/gpphkfbcpidddadnkolkpfckpihlkkil)
+* [Встановити для **Chrome**](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=uk)
+* [Встановити для **Firefox**](https://addons.mozilla.org/uk/firefox/addon/react-devtools/)
+* [Встановити для **Edge**](https://microsoftedge.microsoft.com/addons/detail/react-developer-tools/gpphkfbcpidddadnkolkpfckpihlkkil?hl=uk)
 
-Now, if you visit a website **built with React,** you will see the _Components_ and _Profiler_ panels.
+Тепер під час відвідування вебсайту, **створеного за допомогою React**, ви побачите вкладки _Components_ і _Profiler_.
```

Comment:
```suggestion
Тепер, якщо ви відвідаєте вебсайт, **який створений за допомогою React**, ви побачите вкладки _Components_ і _Profiler_.
```

Reply:
imho, that word-by-word translation is not so good and appropriate in Ukrainian